bookwyrm/migrations/0190_book_search_updates.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.20 on 2023-11-24 17:11
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("bookwyrm", "0188_theme_loads"),
+    ]
+
+    operations = [
+        migrations.RemoveIndex(
+            model_name="author",
+            name="bookwyrm_au_search__b050a8_gin",
+        ),
```

Comment:
I added searching for authors in https://github.com/bookwyrm-social/bookwyrm/pull/3274, but it doesn't use `search_vector`. Should it?

Reply:
I will change author search to use `search_vector` instead of `TrigramSimilarity`; would you remove the `RemoveIndex` here?

Reply:
Interestingly, the `Author` model inherits the `search_vector` field from `BookDataModel`, but there is no trigger that actually fills the `bookwyrm_author.search_vector` column with (for instance) the author name, so this column is empty for every record.

Feel free to leave in this `RemoveIndex` and I'll re-add it in a separate PR that also adds the actual trigger that fills the column and starts using it for author search.

Or if you feel like it, you could add that trigger in this PR.

Reply:
> Feel free to leave in this RemoveIndex and I'll re-add it in a separate PR that also adds the actual trigger that fills the column and starts using it for author search.

Let's do this; thank you!